File: 4-word.cpp
#include<iostream>
#include<iostream>
#include<string>
#include<sstream>
#include<cmath>
#include <algorithm>
using namespace std;


int main(void)
{

	cout << "enter your first string" << endl;
	string x;
	cin >> x;
	int u = 0;
	int l = 0;

	for (int i = 0;i < x.length();i++) {
		if (x[i]>='a'&&x[i]<='z')
		{
			l++;
		}
		else
			u++;
	}
	if (u > l) {
		std::transform(x.begin(), x.end(), x.begin(), ::toupper);
		cout << x << endl;
	}
	if (u < l||u==l)
	{
		std::transform(x.begin(), x.end(), x.begin(), ::tolower);
		cout << x << endl;

	}



}

File: 4-word .cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace excercing
{
    
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("enter your string");
            string x=Console.ReadLine();
            int u = 0;
            int l = 0;
            for(int i=0; i<x.Length; i++)
            {
                if (x[i]>='a'&&x[i]<='z')
                {
                    l++; 
                }
                else
                    u++;
            }
            if (u > l)
            {
                Console.WriteLine(x.ToUpper());
            }
            if (u < l||u==l)
            {
                Console.WriteLine(x.ToLower());
            }
        
            

            



        }
        




    }

    
    


}
